deploy/example/example-app.yaml

```diff
@@ -67,3 +67,6 @@ spec:
                   csi.cert-manager.io/issuer-name: ca-issuer
                   csi.cert-manager.io/dns-names: "${POD_NAME}.${POD_NAMESPACE}.svc.cluster.local"
                   csi.cert-manager.io/uri-sans: "spiffe://cluster.local/ns/${POD_NAMESPACE}/pod/${POD_NAME}}/${POD_UID}"
+                  csi.cert-manager.io/keystore-pkcs12-enable: "true"
```

Comment:
I think a "boolean" enabled attribute can be useful, but could it make sense to make it optional? I think we should look at the API in cert-manager, and try to do something similar here. Over there we have the "create" field.

Reply:
Thinking a bit more, we cannot make it optional, because the default then must be "false", making it a bit less useful. But I think it is nice for the UX to disable this with an attribute, instead of removing the other attributes.

Reply:
The issue we have is that we are working on just a `map[string]string` which is basically just a flat struct of strings. This makes it really tricky when it comes to defaulting, since this is distinct and happens before validation.

Without an explicit bool field, it becomes non-obvious what the behavior of the defaulter and validator should do (and then what ultimately will then get persisted to `metadata.json`)

We would end up basically defaulting PKCS12 fields IFF the password key was defined, and then validating that either both or neither are defined.

Don't feel strongly either way though.

Reply:
👍 I think I prefer the enable flag instead of weird defaulting if another field is set wizardry